keyboards/nix_studio/lilith/rules.mk

```diff
@@ -0,0 +1,2 @@
+# This file intentionally left blank
```

Comment:
This file is blank?

Reply:
should have been blank if it wasnt for chibios. wear leveling is included momentarily as it gets fixed. ill remove the comment in the file